README.md

```diff
@@ -4,20 +4,25 @@
 ### <img align="left" src="logo.svg" width="48">
 # JsonApiX
 
+## Description
+
 - JSON API X is an Android, annotation processor library with the intention of extending it to a KMP library in due time
 - Implements a parser between Kotlin classes and JSON API specification strings in both directions
 - Includes Retrofit module for easy API implementations
 
-### List of libraries used in the project
-
-This chapter lists all the libraries that make a bigger contribution to the project. It is essential for any future contributor to get
-familiar with them, before starting any work on the project. For that reason, links to the documentation are a part of the list.
+## Table of contents
 
-- Kotlinx serialization - [kotlinx.serialization/serialization-guide.md](https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/serialization-guide.md)
-- Kotlin poet - https://square.github.io/kotlinpoet/
+* [Requirements](#requirements)
+* [Getting started](#getting-started)
+* [Usage](#usage)
+* [JsonApiModel - Handling `links` and `meta` JSON API fields](#jsonapimodel---handling-links-and-meta-json-api-fields)
+* [Retrofit Support](#retrofit-support)
+* [List of libraries used in the project](#list-of-libraries-used-in-the-project)
+* [Contributing](#contributing)
+* [License](#license)
+* [Credits](#credits)
 
-Along with the libraries documentation, the developer should also know his way around the JSON API, so here is a link to
-the [specification](https://jsonapi.org/).
+## Requirements
 
```

Comment:
`"minSdk"    : 23,`
But I see all modules are java libs so this doesn't apply.

Can we implement JsonApiX into every java/kotlin project?

Reply:
We don't need to have restrictions on SDK, it's a kotlin/java library, and yes it can work everywhere with java, however It's not KMP yet

We are using kapt, which on java project can be added the processer using `annotationProcessor`

Reply:
@DrakslerT then add this

Reply:
Great